conversion/conversion.go

```diff
@@ -403,10 +408,14 @@ func VerifyDb(ctx context.Context, adminClient *database.DatabaseAdminClient, db
 	return dbExists, err
 }
 
-// CheckExistingDb checks whether the database with dbURI exists or not.
+// CheckExistingDb checks whether the database with dbURI exists or not. If request is not completed within
+// context deadline then it will return an context error.
 func CheckExistingDb(ctx context.Context, adminClient *database.DatabaseAdminClient, dbURI string) (bool, error) {
 	_, err := adminClient.GetDatabase(ctx, &adminpb.GetDatabaseRequest{Name: dbURI})
 	if err != nil {
+		if err == ctx.Err() {
```

Comment:
better to check for some specific part of error msg rather than comparing the entire err

Reply:
The error which is returned is ctx.Err() -> context deadline exceeded. This comparison won't take much time. 
If we don't compare the full error then doesn't it compromise the correctness?

Reply:
in that case we should compare ctx.err with context deadline exceeded not err

Reply:
The value of ctx.err is 'context deadline exceeded' if context expires before API call is completed. So we should compare the error returned with ctx.err, right?